Installation/lib/cmake/CGAL/CGALConfig.cmake

```diff
@@ -62,9 +62,9 @@ else()
   endif()
 endif()
 
-message(STATUS "******************************************** CGAL Config.cmake")
-
 include(${CGAL_MODULES_DIR}/CGAL_CreateSingleSourceCGALProgram.cmake)
+include(${CGAL_MODULES_DIR}/CGAL_Macros.cmake)
```

Comment:
`CGAL_Macro` is really a module I would like to remove from our CMake scripts. Where is it needed?

Reply:
It was already there before; I just moved it to regroup the three include at the same place.

Reply:
Right. :-(

Reply:
We will need further clean up steps later...